pkg/exchange/coinbase/stream_handlers.go

```diff
@@ -300,7 +318,13 @@ func (s *Stream) handleMatchMessage(msg *MatchMessage) {
 		return
 	}
 	trade := msg.Trade()
-	s.EmitTradeUpdate(trade)
+	if _, fullExist := s.subLocalChannelsMap[fullChannel]; fullExist {
```

Comment:
Why not use isPublicOnly

Reply:
I want it to be more explicit.
Though `.PublicOnly` does imply `fullExist`.

Reply:
After a second thought, I think you are right.
The logic is simpler to check on `.PublicOnly`.
It's fixed.